index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link href="assets/css/style.css" rel="stylesheet" type="text/css">
+
+    <title>Codecademy Community</title>
+</head>
+<body>
+    <header>
+        <img class="navlogo" src="assets/img/codecademy_logo.png">
+
+        <nav>
+            <li class="navlink"><a href="./home.html">Home</a></li>
+            <li class="navlink"><a href="./staff.html">Server Staff</a></li>
+            <li class="navlink"><a href="./contact.html">Contact Us</a></li>
+            <li class="navlink"><a href="./sharing_code.html">Get Help</a></li>
+        </nav>
+    </header>
+    <main>
+        <section class="serverdescription">
+            <h2>Codecademy Community</h2>
+            <p class="descriptiontext">
+                Welcome to the Codecademy Discord Community! This is the website of the official
+                Codecademy Discord, run by the moderator team from the forums as well as users
+                from the community. Our goal is to create a friendly place for all to converse,
```

Comment:
Rather than "run by the moderator team from the forums as well as users from the community", possibly change it to "run by volunteer moderators and community members".